### DISCUSSION TITLE
Integrate Aqua checksum update into renovate?

### DISCUSSION THREAD

**Discussion by sheldonhull** [aquaproj/aqua#2791] [Deprecated (Question and Answer)] [answered]
You ever figure out if possible to integrate Aqua checksum update into the renovate update?

https://docs.renovatebot.com/configuration-options/#postupgradetasks

I was considering trying to have to run through postupgrade option but uncertain if maybe you'd tried that and it didn't work. I'm using self hosted runs for renovate so might work. 

As it stands Aqua update PRs break the Aqua installs because I didn't create a self hosted Aqua checksum update action. Also if I did it would break renovate updating that branch so I was trying to do all in the same workflow.

**Comment by suzuki-shunsuke** [comment, marked as answer]
I don't use self-hosted Renovate, and Post-upgrade tasks can only be used on self-hosted Renovate instances, so I have never tried Post-upgrade tasks.
As you said, I guess you can update aqua-checksums.json via Post-upgrade tasks by executing `aqua upc` command.
Of course, you have to install aqua in self-hosted Renovate instances.
